requirements/requirements.txt

```diff
@@ -10,5 +15,7 @@ tensorboard
 text-unidecode
 torch
 tqdm>=4.41.0
+transformer-engine @ git+https://github.com/NVIDIA/TransformerEngine.git@${TE_TAG}
+unstructured==0.14.9
```

Comment:
Not a core requirement. Its for processing unstructured data - https://pypi.org/project/unstructured/

Reply:
Makes sense. But which file is the best place for it?